src/main/java/org/gridsuite/cgmes/assembling/job/CgmesUtils.java

```diff
@@ -22,24 +22,29 @@
  * @author Franck Lecuyer <franck.lecuyer at rte-france.com>
  */
 public final class CgmesUtils {
-
     private static final Logger LOGGER = LoggerFactory.getLogger(CgmesUtils.class);
-
-    // We add sourcingActor "XX" for test purpose
-    private static final Set<String> NEEDED_SOURCING_ACTORS = new TreeSet<>(Arrays.asList("REE", "REN", "RTEFRANCE", "REE-ES", "REN-PT", "RTEFRANCE-FR", "ELIA", "TTN", "XX"));
-
     private static final String SV_MODEL_PART = "SV";
     private static final String EQ_MODEL_PART = "EQ";
     private static final String SSH_MODEL_PART = "SSH";
     private static final String TP_MODEL_PART = "TP";
-
     private static final Set<String> NEEDED_PROFILES = new TreeSet<>(Arrays.asList(EQ_MODEL_PART, SSH_MODEL_PART, SV_MODEL_PART, TP_MODEL_PART));
+    private static final String DOT_REGEX = "\\.";
+    private static final String UNDERSCORE_REGEX = "\\_";
```

Comment:
_ is not a special regex character, and should not be escaped

Reply:
Done